octoprint_mrbeam/iobeam/iobeam_handler.py

```diff
@@ -485,13 +485,14 @@ def _work(self):
                         data = None
                         try:
                             sock_data = self._my_socket.recv(self.MESSAGE_LENGTH_MAX)
-                            if len(temp_buffer) > 0:
-                                self._logger.warn(
-                                    "ANDYTEST temp_buffer: %s", len(temp_buffer)
-                                )
-                            if len(sock_data) > self.MESSAGE_LENGTH_MAX - 6:
-                                self._logger.warn(
-                                    "ANDYTEST sock_data: %s", len(sock_data)
+                            if (
+                                len(temp_buffer) >= self.MESSAGE_LENGTH_MAX
+                                or len(sock_data) >= self.MESSAGE_LENGTH_MAX - 5
+                            ):
+                                self._logger.info(
+                                    "Receiving long message - buffer size: %s, receiving size : %s",
+                                    len(temp_buffer),
+                                    len(sock_data),
```

Comment:
I think we can remove this completely. Let's quickly discuss a potential socket saturation/overflow.

Reply:
I agree.

Reply:
pls remove the logging